modules/hal_silabs/simplicity_sdk/src/sl_memory_manager_shim.c

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2024 Silicon Laboratories Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Shim basic sl_* allocation functions to libc malloc, which again
+ * will get redirected to the Zephyr sys_heap.
+ */
+
+#include "sl_memory_manager.h"
+#include <stdlib.h>
```

Comment:
this is not really zephyr glue code, as this uses stdc lib

Reply:
> this is not really zephyr glue code, as this uses stdc lib

It's code that's not part of any of our SDKs, so it needs to sit somewhere. It's possible this might eventually be updated to use the system heap rather than the libc heap.